pkgs/applications/audio/exact-audio-copy/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, fetchurl
+, makeDesktopItem
+, imagemagick
+, p7zip
+, wine
+, writeShellScriptBin
+, symlinkJoin
+, use64 ? false
+}:
+
+let
+  pname = "exact-audio-copy";
+
+  eac_exe = fetchurl {
+    url = "http://www.exactaudiocopy.de/eac-1.6.exe";
+    sha256 = "8291d33104ebab2619ba8d85744083e241330a286f5bd7d54c7b0eb08f2b84c1";
+  };
+
+  cygwin_dll = fetchurl {
+    url = "https://cygwin.com/snapshots/x86/cygwin1-20220301.dll.xz";
+    sha256 = "0zxn0r5q69fhciy0mrplhxj1hxwy3sq4k1wdy6n6kyassm4zyz1x";
+  };
+
+  patched_eac = stdenv.mkDerivation {
+    name = "patched_eac";
```

Comment:
```suggestion
    pname = "patched_eac";
    version = "...";
```

Reply:
I'm assuming I should replace the "..." with the the version I actually want "1.6.0", right? Like there's nothing special about the ellipsis?

Reply:
Yes, we should set some version number.

Reply:
Gotcha, updated to 1.6.0